packages/solutions/app-tools/src/builder/shared/builderPlugins/adapterSSR.ts

```diff
@@ -234,7 +234,12 @@ function applySSRDataLoader<B extends Bundler>(
 
   const { entriesDir = './src' } = normalizedConfig.source;
 
-  const absolutePath = path.resolve(appDirectory, entriesDir);
+  // Both match in Windows and Unix(Linux, Mac)
+  // docs: https://github.com/webpack/webpack/issues/2073
+  const absolutePath = path
+    .resolve(appDirectory, entriesDir)
+    .split(path.sep)
+    .join('(\\\\|/)');
```

Comment:
How about using the `normalizeToPosixPath` util

Reply:
In template, we will product a loader path like this:
```js
import loader_0 from 'C:/a/b/c.loader.ts'
```
We need match this path use data-loader.
If we transform the path by using `normalizeToPosixPath`, also need  transform the template path. Like this:
```js
import loader_0 from '/C/a/b/c.loader.ts'
```

But if we use /C/a/b/c.loader.ts to match real files, wouldn't the windows system be able to find the file?